content/manuals/scout/policy/_index.md

```diff
@@ -146,8 +147,8 @@ The list includes the following vulnerabilities:
 - [CVE-2023-44487 (HTTP/2 Rapid Reset)](https://scout.docker.com/v/CVE-2023-44487)
 - [CVE-2024-3094 (XZ backdoor)](https://scout.docker.com/v/CVE-2024-3094)
 
-You can configure the CVEs included in this list by creating a custom policy.
-Custom configuration options include:
+You can customize this policy to change which CVEs that are considered
+high-profile by configuring the policy. Custom configuration options include:
 
 - **CVEs to avoid**: Specify the CVEs that you want to avoid in your artifacts.
 
```

Comment:
The default is actually "none" as in _no_ CVEs are excluded from the built-in list of high-profile CVEs. 

Note that from time to time we will be adding to the list. Do we want to update the docs each time that happens? Above we say "includes the following" which means we don't **have** to do this.